Assets/Scripts/Player.cs

```diff
@@ -0,0 +1,77 @@
+﻿using UnityEngine;
+
+public class Player : MonoBehaviour
+{
+    [SerializeField] private float speed;
+    [SerializeField] private float jumpForce;
+    [SerializeField] private GameObject body;
+
+    private IUnityService unityService;

```

Comment:
If we will follow my suggestions about separating interfaces, I would change it to
```suggestion
    private IInputService inputService;
    private ITimeService timeService;
```